src/main/java/io/zeebe/containers/ZeebeBrokerContainer.java

```diff
@@ -157,4 +158,13 @@ public ZeebeBrokerContainer withEmbeddedGateway(final boolean embedGateway) {
     this.embedGateway = embedGateway;
     return withEnv(ZeebeBrokerEnvironment.EMBED_GATEWAY, embedGateway);
   }
+
+  /**
+   * Attempts to stop the container gracefully. If it times out, the container is abruptly killed.
+   *
+   * @param timeout must be greater than 1 second
```

Comment:
Can we add also that to kill the container we should use `#stop`?

Reply:
Isn't that implicit by naming `gracefullyShutdown`? Also, the fact that stop actually kills the container is a particularity of another API, isn't is strange to document it in ours?

Reply:
Hm, in that sense yes. I guess I was anticipating users to find the two APIs confusing (since from a UX point of view, both methods are callable from the same instance). Maybe we can specify that it shutdowns gracefully the container but does not remove/kill it at least.